### ISSUE TITLE
Parameters of forced convection

### ISSUE THREAD

**Issue by christoph-maurer** [LBNL-ETA/pyWinCalc#46]
The example [gaps_and_gases](https://github.com/LBNL-ETA/pyWinCalc/blob/main/examples/gaps_and_gases.py) works well! When I change the gap `thickness` or the `forced_ventilation_air_speed` or the `forced_ventilation_air_temperature` in [line 79](https://github.com/LBNL-ETA/pyWinCalc/blob/e7b3ae9c198bebf12007c7669eb2f84858c9cbca/examples/gaps_and_gases.py#L79), it doesn't stop to calculate. Do you know why?

**Comment by vidanovic** [comment]
@christoph-maurer Does it happen if you put any other number? I can take a look today. The only reason I can think of is convergence problem, but in case of forced ventilation it does not make sense since convection speed is known.

**Comment by christoph-maurer** [comment]
@vidanovic Yes, it looks like a convergence problem. In the tests `https://github.com/LBNL-ETA/Windows-CalcEngine/tree/main/src/Tarcog/tst/units/GapBetween*`, an `airGapSpeed` of 0, 0.1, 0.5 and 1 converged.
